great_expectations/core/batch.py

```diff
@@ -871,50 +871,39 @@ def get_batch_request_from_acceptable_arguments(
 def standardize_batch_request_display_ordering(
     batch_request: Dict[str, Union[str, int, Dict[str, Any]]]
 ) -> Dict[str, Union[str, Dict[str, Any]]]:
-    datasource_name: str = batch_request["datasource_name"]
-    data_connector_name: str = batch_request["data_connector_name"]
-    data_asset_name: str = batch_request["data_asset_name"]
+    datasource_name: str = batch_request.get("datasource_name")
+    data_connector_name: str = batch_request.get("data_connector_name")
+    data_asset_name: str = batch_request.get("data_asset_name")
```

Comment:
Since these are nullable values, can we please annotate with `Optional`?

Reply:
✅